core/pom.xml

```diff
@@ -14,7 +14,7 @@
 
    <properties>
       <version.log4j>1.2.16</version.log4j>
-      <version.log4j2>2.3</version.log4j2>
+      <version.log4j2>2.8.2</version.log4j2>
```

Comment:
2.11.0 seems to the the latest version - https://logging.apache.org/log4j/2.x/download.html

Reply:
https://github.com/radargun/radargun/issues/525